plugins/modules/gitlab_group.py

```diff
@@ -279,10 +286,13 @@ def update_group(self, group, arguments):
 
         return (changed, group)
 
-    def delete_group(self):
+    '''
+    @param force To delete even if projects inside
+    '''
+    def delete_group(self, force=False):
```

Comment:
In Python, docstrings are usually the first string inside a function:
```suggestion
    def delete_group(self, force=False):
        '''
        @param force To delete even if projects inside
        '''
```
The remainder of this Python file does it different though...

Reply:
Yes exactly, that's why i set arguments outside function like others functions.